frontend/src/pages/newroom.tsx

```diff
@@ -9,7 +9,11 @@ const NewRoom: React.FC<RouteChildrenProps<any>> = (props) => {
     const handleSubmit = () => {
         API.post('/api/room', { title: roomName })
             .then((res) => {
-                props.history.push(`/room/${res.data.data.id}`);
+                if (roomName.trim().length > 0) {
+                    props.history.push(`/room/${res.data.data.id}`);
+                } else {
+                    alert('Empty room name is not permitted');
+                }
```

Comment:
I didn't realize last time but this is logically wrong.
Here we are creating a room then denying from joining if it's a blank string. 

better would be:
- To do this validation before submitting, and if blank string, do not make a post request
- Add same validation in joinroom:
    - In header:  instead of `if (roomId)` do `if (roomId.trim())`
    - similaryly in joinroom page

Reply:
Okay sure, I'll fix it